cli/src/main.rs

```diff
@@ -62,11 +67,15 @@ async fn main() -> color_eyre::Result<()> {
             allow_existing,
         } => {
```

Comment:
variables `user_id,`` raw_html`, `url_parsed`, and `cleaned_html` are intermediate steps required to process the URL, clean its HTML content, and store it in the database